Defs/ThingDefs_Misc/Apparel_Various.xml

```diff
@@ -48,51 +48,6 @@
 
   <!-- ======================================= Armor vests =======================================-->
 
```

Comment:
Why remove the flak vest?

Reply:
Vanilla added a flak vest

Reply:
Vanilla renamed the plate vest, we're not following that change. Should keep CE vests as is and only update defNames to account for vanilla rename.